test/integration/full/test-webdriver.js

```diff
@@ -0,0 +1,236 @@
+/*global window, Promise */
```

Comment:
This is more or less a copy paste of the grunt script by the same name, minus all the grunt things and with the ability to configure chrome and chromedriver